src/Raven.Studio/wwwroot/Content/css/virtual-grid.less

```diff
@@ -0,0 +1,79 @@
+﻿.virtual-grid {
+	overflow-x: auto;
+	overflow-y: hidden;
+	position: absolute;
+	top: 0;
+	bottom: 0;
+	left: 0;
+	right: 0;
+
+    @columnHeaderHeight: 30px;
+    .column-container {
+	    white-space: nowrap;
+	    height: @columnHeaderHeight;
+    }
+
+    .viewport {
+	    position: absolute;
+        top: 0;
+	    left: 0;
+	    right: 0;
+	    bottom: 0;
+	    overflow-y: auto;
+	    overflow-x: hidden;
+
+        &.columns-visible {
+            top: @columnHeaderHeight;
+        }
+    }
+
+    .viewport-scroller {
+	    position: relative;
+    }
+
+    .column {
+	    display: inline-block;
+	    text-overflow: ellipsis;
+	    overflow: hidden;
+	    text-align: left;
+	    border-bottom: 2px solid rgb(221, 221, 221);
```

Comment:
we should avoid specifing colors directly, @kwiato will change this to variables, so we can later support color themes.